src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1239,8 +1261,9 @@ public static function getMimeTypes(string $format): array
      * @param string|null $mimeType        The mime type to check
      * @param bool        $subtypeFallback Whether to fall back to the subtype if no exact match is found
      */
-    public function getFormat(?string $mimeType, bool $subtypeFallback = false): ?string
+    public function getFormat(?string $mimeType/* , bool $subtypeFallback = false */): ?string
     {
+        $subtypeFallback = 2 <= \func_num_args() ? func_get_arg(1) : false;
```

Comment:
No need to trigger a deprecation that the second parameter will be required in 8.0?

Reply:
We rely on DebugClassLoader for that